backend/allowed-licenses.json

```diff
@@ -0,0 +1,5 @@
+[
+    "MIT",
```

Comment:
Are We sure those are the only licenses We want to be using? What about BSD ones?

Reply:
Of course it is not all licenses we will be using. I have added all licenses that are currenlty used by dependencies in our project. If there will be a new dependency with a new license then it should be investigated and added to this list, if the decsion will be it is a safe license.

Reply:
I think that We do need a list of licensees which We are comfortable using to begin with, otherwise what is the point of a license checker? We might then latter on append additional stuff to it if We find a library with a license which We did not know about but are comfortable using.

To summarize I think that We should expand this list to include all licenses which We think We would be comfortable with before We merge in this.

Reply:
Synchronized both lists for checkers. I think right now we have a good starting point with licenses.